tests/js/spec/views/replays/detail/console/messageFormatter.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {
+  BreadcrumbLevelType,
+  BreadcrumbType,
+  BreadcrumbTypeDefault,
+} from 'sentry/types/breadcrumbs';
+import {MessageFormatter} from 'sentry/views/replays/detail/console/consoleMessage';
+
+const breadcrumbs: BreadcrumbTypeDefault[] = [
+  {
+    type: BreadcrumbType.DEBUG,
+    category: 'console',
+    data: {
+      arguments: ['This is a %s', 'test'],
+      logger: 'console',
+    },
+    level: BreadcrumbLevelType.LOG,
+    message: 'This is a %s test',
+    timestamp: '2022-06-22T20:00:39.959Z',
+  },
+  {
+    type: BreadcrumbType.DEBUG,
+    category: 'console',
+    data: {
+      arguments: ['test', 1, false, {}],
+      logger: 'console',
+    },
+    level: BreadcrumbLevelType.LOG,
+    message: 'test 1 false [object Object]',
+    timestamp: '2022-06-22T16:49:11.198Z',
+  },
+  {
+    type: BreadcrumbType.DEBUG,
+    category: 'console',
+    data: {
+      arguments: [{}],
+      logger: 'console',
+    },
+    level: BreadcrumbLevelType.ERROR,
+    message: 'Error: this is my error message',
+    timestamp: '2022-06-22T20:00:39.958Z',
+  },
```

Comment:
```suggestion
  {
    type: BreadcrumbType.DEBUG,
    category: 'console',
    data: {
      arguments: [{}],
      logger: 'console',
    },
    level: BreadcrumbLevelType.ERROR,
    message: 'Error: this is my error message',
    timestamp: '2022-06-22T20:00:39.958Z',
  },
```

Can we add a test case for `log('test', ['foo', 'bar'])`